### DISCUSSION TITLE
A problem with PML

### DISCUSSION THREAD

**Discussion by wangrongqian2019** [devitocodes/devito#1681] [Q&A]
When I use pml method(https://nbviewer.jupyter.org/github/devitocodes/devito/blob/master/examples/seismic/abc_methods/03_pml.ipynb) on the marmousi model(all the setting is same to the example on the Jupyter notebook), the solution seems to blow up, could anyone help me?
<img width="689" alt="d36040250a2a6de50daea5ed1f70bd9" src="https://user-images.githubusercontent.com/54973121/115992533-c9283b80-a600-11eb-905b-3917cc6c865b.png">

**Comment by wangrongqian2019** [comment]
I has already been fixed this problem now.
I made a mistake on coding.

**Reply by jiwei08** [reply]
老哥稳
